test/src/test/java/org/corfudb/infrastructure/logreplication/LogReplicationUtilsTest.java

```diff
@@ -1,7 +1,9 @@
 package org.corfudb.infrastructure.logreplication;
 
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
 import com.google.protobuf.Message;
 import lombok.extern.slf4j.Slf4j;
+import org.corfudb.integration.LogReplicationListenerIT;
```

Comment:
Why does the IT get imported?

Reply:
It is used to get the class name when setting the name for the thread executor